src/azure-friday.function/.deployment

```diff
@@ -0,0 +1,3 @@
+[config]
```

Comment:
This is a temporary file that is here due to a bug in azd.  This can be removed after the next azd release.